categories/math.json

```
@@ -1,5 +1,5 @@
 {
   "$schema": "../category.schema.json",
-  "title": "Maths",
+  "title": "Math",
```

Maybe let's use a more universal name?
```suggestion
  "title": "Mathematics",
```

---

Good idea!